packages/domain/src/use-cases/inbox/GetAllConversations.ts

```diff
@@ -0,0 +1,31 @@
+import { EthereumAddress, Result } from '@lens-protocol/shared-kernel';
+
+import { ProfileId } from '../../entities';
+import { Conversation, ConversationsDisabledError } from '../../entities/Conversation';
+
+export type GetAllConversationsRequest = {
+  profileId?: ProfileId;
+  address: EthereumAddress;
```

Comment:
Do you think we still need `address` here with this new flow? Should it use the Active Wallet?

To understand this I am trying to entertain the idea of what happens if the user logs out (with `useWalletLogin`) and logs back in with another Signer/Wallet. 🤔

Reply:
in some other hooks, we always require activeProfile (i.e. in useFollow as `follower`) but we could also just read it internally in the hook.
so I just followed the same pattern. when the Signer changes, then the address provided to `useConversations` hook will also change as it is taken from `useActiveWallet` hook. 
at least that's what happens in the example in the web-wagmi app.

Reply:
But Active Wallet and Active Profile are significantly different things.

Active Wallet determine the EOA used to login. Active Profile is one among several Lens Profile that can be owned by the Active Wallet EOA.

We do not bind ourself to the user's wallet like many crypto-oriented dApps do. This is on purpose so that after initial signing the social app based on the Lens SDK feels like a web2 experience. User can switch the network and/or account on their wallet app and there should not be consequence on their social app based  on the Lens SDK.

Only when a signature is needed the SDK requires a network switch and verifies the EOA address to match the Active Wallet address.

This is a design decision.

So the idea of Signer changing dynamically is not really a problem for the SDK. For the SDK the Active Wallet changes as consequence of logout or login with another wallet.

Reply:
ok, done